src/environments/environment.staging.ts

```diff
@@ -1,4 +1,12 @@
 export const environment = {
   production: true,
   base_url: 'https://mobile-content-api-stage.cru.org/',
+  oidc_auth: {
+    client_id: '0oa1ll7gg0vVRLctz0h8',
+    client_secret: '738c0f49-b689-437a-9920-4f4f1fbfb7bb',
```

Comment:
@frett won't we want a separate app inside of Okta for each stage and prod?

Reply:
I actually don't think we will need a separate app. The application only provides authentication, the API itself enforces authorization.